Octokit.Reactive/Clients/ObservableDeploymentsClient.cs

```diff
@@ -31,11 +31,28 @@ public ObservableDeploymentsClient(IGitHubClient client)
         /// <returns>All the <see cref="Deployment"/>s for the specified repository.</returns>
         public IObservable<Deployment> GetAll(string owner, string name)
         {
+            return GetAll(owner, name, ApiOptions.None);
```

Comment:
Like we mentioned earlier, can you bring back the `Ensure` arguments for `owner` and `name` here?